src/platform/3ds/CMakeLists.txt

```diff
@@ -38,6 +52,23 @@ add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/font.c
                    COMMAND ${RAW2C} ${CMAKE_SOURCE_DIR}/src/platform/3ds/font.raw
                    DEPENDS ${CMAKE_SOURCE_DIR}/src/platform/3ds/font.raw)
 
+add_custom_command(
+	OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/uishader.shbin ${CMAKE_CURRENT_BINARY_DIR}/uishader.shbin.h
+	MAIN_DEPENDENCY ${CMAKE_CURRENT_SOURCE_DIR}/uishader.vsh
+	COMMAND ${PICASSO}
+		-o ${CMAKE_CURRENT_BINARY_DIR}/uishader.shbin
+		-h ${CMAKE_CURRENT_BINARY_DIR}/uishader.shbin.h
+		${CMAKE_CURRENT_SOURCE_DIR}/uishader.vsh
+	COMMENT "picasso uishader.vsh"
+	)
```

Comment:
This should be a custom target. I can fix that after a merge if you don't want to deal with CMake though. Ditto about the paren though.